.docker/parameters.yml

```diff
@@ -16,8 +16,10 @@ parameters:
     gtm_id: ~
     cookie_consent: false
     trusted_hosts:
-      - localhost
-      - web
+      - .*\.localhost$
+      - ^localhost$
+      - .*\.?web$
+      - ^web$
```

Comment:
Why are these changing?

Reply:
Suspect related to https://github.com/elifesciences/journal/pull/1270#discussion_r424405752.

Reply:
Really `submit_url` should become a list instead?

Reply:
from discussing with @erkannt we think it should be a regex: ^.*\.elifesciences.org$

I will use a different setting